spec/features/merchants/items/index_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe "Merchants items index" do
+  # 6. Merchant Items Index Page
```

Comment:
I think we should delete the commented out user story before pushing. Great to use while writing though.

Reply:
Done!